src/app/index.config.coffee

```diff
@@ -8,7 +8,7 @@ angular.module 'mnoEnterpriseAngular'
     # Set options third-party lib
     toastrConfig.timeOut = 5000
     toastrConfig.positionClass = 'toast-top-right'
-    toastrConfig.preventDuplicates = true
+    toastrConfig.preventDuplicates = false
```

Comment:
Obviously nooooox :P But yeah it's not super crucial, but I was using it in Transac!